lib/rules/disallow-capitalized-comments.js

```diff
@@ -57,7 +57,7 @@ module.exports.prototype = {
         var letterPattern = require('../../patterns/L');
         var lowerCasePattern = require('../../patterns/Ll');
 
-        file.getComments().forEach(function(comment) {
+        file.iterateTokensByType(['Line', 'Block'], function(comment) {
```

Comment:
Actually, we don't need to. It's not a part of Parser API because comments doesn't affect code execution and they are just skipped on parsing for execution. Since we merging comment tokens into code ASD we already using superset of Parser API and `Line` and `Block` types are confusing. I suggest to fix that and to describe it in JSFile docs.